spotify-track-search.el

```diff
@@ -130,4 +131,20 @@ be played in the context of its album."
     (tabulated-list-init-header)
     (tabulated-list-print t)))
 
+
```

Comment:
```suggestion
```

I'd remove this line to keep only one blank line separating each function definition.